cmd/destroy/destroy_test.go

```diff
@@ -978,3 +978,51 @@ func TestDestroyWithForceOptionAndFailedCommands(t *testing.T) {
 	cfg, _ := configmaps.Get(ctx, pipeline.TranslatePipelineName(opts.Name), okteto.Context().Namespace, fakeClient)
 	assert.Nil(t, cfg)
 }
+
+func TestShouldRunInRemoteDestroy(t *testing.T) {
```

Comment:
Could we add test cases in both tests about `shouldRunInRemote(opts)` (in deploy and remote) function where we set `OKTETO_DEPLOY_REMOTE` and `OKTETO_FORCE_REMOTE` using `t.Setenv`?